event_collector/handler.py

```diff
@@ -25,7 +25,12 @@
     failed_elements_response,
     ok_response,
 )
-from event_collector.metadata_api_client import MetadataApiClient, ServerErrorException
+from event_collector.metadata_api_client import (
+    MetadataApiClient,
+    ServerErrorException,
+    version_exists,
+    get_confidentiality,
```

Comment:
Bør disse to flyttes ut i en egen modul? Da jeg leste `from metadata_api_client import ...` første gang så tenkte jeg i hvert fall at de fortsatt kommuniserer med API-et.

Reply:
Renamet modulen til bare `event_collector.metadata` - tenker det funker bedre?

Reply:
Enig. :+1: